odoo/addons/base/models/ir_fields.py

```diff
@@ -42,6 +42,26 @@ def _format_import_error(self, error_type, error_msg, error_params=(), error_arg
                 error_params = tuple(sanitize(v) for v in error_params)
         return error_type(error_msg % error_params, error_args)
 
+    def _get_import_field_path(self, field, value):
+        """ Rebuild field path for import error attribution to the right field.
+        This method uses the 'parent_fields_hierarchy' context key built during treatment of one2many fields
+        (_str_to_one2many). As the field to import is the last of the chain (child_id/child_id2/field_to_import),
+        we need to retrieve the complete hierarchy in case of error in order to assign the error to the correct
+        column in the import UI.
+        """
+        field_path = [field]
+        parent_fields_hierarchy = self._context.get('parent_fields_hierarchy')
+        if parent_fields_hierarchy:
+            field_path = parent_fields_hierarchy + field_path
+
+        field_path_value = value
+        while isinstance(field_path_value, list):
+            key = list(field_path_value[0].keys())[0]
+            if key:
+                field_path.append(key)
+            field_path_value = field_path_value[0][key]
```

Comment:
Impossible to know what this does without a structure example in the docstring for "value".